cinema/serializers.py

```diff
@@ -125,9 +125,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
```

Comment:
Ensure that the keys 'row', 'seat', and 'movie_session' exist in `attrs` before accessing them to prevent potential `KeyError`. Consider using `attrs.get('key')` with default values or adding a check.